### PR TITLE
Added support for order_meta

### PR DESCRIPTION
**Pull request by polystyreneMan**
Added support for order_meta, which is available on PUT, but isn't in the API docs

### CODE REVIEW

File: WooCommerce/Order.cs [hunk 1]
```diff
@@ -212,6 +212,12 @@ public class Order
         /// </summary>
         [DataMember(EmitDefaultValue = false)]
         public Customer customer { get; set; }
+
+        /// <summary>
+        /// Dictionary of order metadata/custom fields. Undocumented.
```
Comment (XiaoFaye) [inline WooCommerce/Order.cs]: Would you please mention in the comment that the order metadata/custom fields when only return when filter[meta] set to true? Thanks.

File: RestAPI.cs [hunk 2]
```diff
@@ -175,7 +175,8 @@ public static string SerializeJSon<T>(T t)
         {
             DataContractJsonSerializerSettings settings = new DataContractJsonSerializerSettings()
             {
-                DateTimeFormat = new DateTimeFormat("yyyy-MM-ddTHH:mm:ssZ")
+                DateTimeFormat = new DateTimeFormat("yyyy-MM-ddTHH:mm:ssZ"),
+                UseSimpleDictionaryFormat = true
```
Comment (XiaoFaye) [inline RestAPI.cs]: I assume that we need to set UseSimpleDictionaryFormat to true in DeserializeJSon function as well?

**Comment by polystyreneMan** [comment]
Ahhh, thanks for pointing that out. I didn't know why order_meta wasn't included on GET, but I didn't need so didn't spend any time on it. I've updated the comment and enabled SimpleDictionaryFormat for deserialization as well. Thanks!

**Comment by XiaoFaye** [comment]
Looks all good, thanks for your work. :)

**Comment by polystyreneMan** [comment]
No problem. Thanks for putting the library together to begin with. Very helpful.

**Comment by MakarkinPRO** [comment]
Is it possible to get order's item **variations**?

**Comment by XiaoFaye** [comment]
I think it's already there, you didn't get it? @MakarkinPRO

**Pull request merged**